sass/components/ClusterOverview/consumers.scss

```diff
@@ -1,10 +1,3 @@
-.kubevirt-consumers__bar .progress {
-  height: 10px;
-  margin-top: 15px;
-}
-
-.kubevirt-consumers__filters {
-  display: flex;
-  padding-bottom: 10px;
-  border-bottom: 1px solid #d1d1d1;
+.kubevirt-consumers__dropdown-filter {
+  width: 49%;
```

Comment:
I think margin should be more descriptive/generic than width

Reply:
Im using Columns + padding as I had some issues on firefox when not using width.